tests/sqlparser_snowflake.rs

```diff
@@ -4438,3 +4438,63 @@ fn test_snowflake_identifier_function() {
         true
     );
 }
+
+#[test]
```

Comment:
Can we merge all the added tests into the same `fn test_create_database()` function? since they are part of the same feature